components/web/firefox/patches/07-branding.patch

```diff
@@ -1,11 +0,0 @@
---- firefox-52.1.0esr/browser/branding/official/pref/firefox-branding.js.~1~	2017-05-04 11:51:32.593399439 +0300
```

Comment:
Please, restore this patch in one or another way - we want to display welcome page on first FF run.